### ISSUE TITLE
Adding reactions to messages will crash the client

### ISSUE THREAD

**Issue by yutamago** [Discord-Net-Labs/Discord.Net-Labs#152]
Adding reactions to messages will crash the client.
The first reaction will work fine.
After the first reaction, no new commands will be received until the client disconnects at some point.

It's still working in Discord.Net 2.4.0, so it must be a change in Discord.Net.Labs.
Tested with Discord.Net.Labs 3.0.4.

Code to reproduce the error:
```cs
public class SomeModule : ModuleBase<SocketCommandContext>
{

    [Command("command1")]
    public async Task command1() {
        await Context.Message.AddReactionAsync(new Emoji("📨")); // this one will show up
        await Context.Message.AddReactionAsync(new Emoji("🕵️")); // this one wont
    }

    [Command("command2")]
    public async Task command2() {
        await Context.Message.AddReactionsAsync(new IEmote[] {
            new Emoji("📨"), // this one will show up
            new Emoji("🕵️")  // this one wont
        });
    }

}
```

**Comment by SylveonDeko** [comment]
Known issue with newest version due to wacky internal rate limiting, downgrade to 3.0.3 for the time being

**Comment by quinchs** [comment]
Fixed in 3.1.0 release
